src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
-// write your code here
+const numbersArray = [...document.querySelectorAll('.population')];
+let tot = 0;
+
+numbersArray.forEach((el) => (tot += parseFloat(el.innerText.replace(/[^0-9.-]+/g, ''))));
```

Comment:
Consider adding error handling for `parseFloat` to manage cases where the text cannot be converted to a number. This will help avoid NaN values in your calculations.